Tests/runtest.py

```diff
@@ -66,9 +66,11 @@
                 print(" %s should be: %s" % (value, expected_value))
                 sys.exit(1)
 
+        """
```

Comment:
I see.  Lets just remove this block completely, instead of pushing something that is commented out.

If you want to add this test, feel free to file a separate pull request - we can discuss there how to get it to work.  For example, we could use OrderedDict instead of dict wherever needed.

Other then this, the current diff looks great, thanks for this work.